deepchem/models/tensorgraph/layers.py

```diff
@@ -480,6 +480,55 @@ def _get_scope_name(self):
       return self._shared_with._get_scope_name()
 
 
+class Highway(Layer):
```

Comment:
Could you add a description of Highway layers and why they're useful here?

Reply:
Just added. I feel that Highway layer is like a simplified version of GRU, I implemented it here to repeat the work in the ORGAN paper, though I have not done a detailed comparison.